src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,57 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        File fileFrom = new File(fromFileName);
+        int buy = 0;
+        int supply = 0;
+        int result;
+        StringBuilder builder = new StringBuilder();
+
+        try {
```

Comment:
This method is pretty huge - you might want to split it into smaller sub-processes, for example file reading part could be externalized as a method accepting `fromFileName` and returning the read string.

Reply:
Fixed.